### PR TITLE
add padding for notes with no modifiers

### PR DESCRIPTION
**Pull request by AaronDavidNewman**
so the heads don't get squished.  Before:
image

After:
image

That's really it! Otherwise, the min size of the note is only the width of the noteheads.

### CODE REVIEW

**Comment by sschmidTU** [comment]
Awesome!
Did you run the visual regression tests with this? It doesn't cause massive changes, hopefully?

File: src/stavenote.ts [hunk 1]
```diff
@@ -121,6 +121,10 @@ export class StaveNote extends StemmableNote {
   static get DEFAULT_LEDGER_LINE_OFFSET(): number {
     return 3;
   }
+  static get minNoteheadPadding() {
```
Comment (rvilarl) [inline src/stavenote.ts:124]: The return value is missing, make it:
`static get minNoteheadPadding(): number`

Comment (AaronDavidNewman) [reply src/stavenote.ts]: Right!

**Comment by rvilarl** [comment]
> Did you run the visual regression tests with this? It doesn't cause massive changes, hopefully?
As commented in #1078, I have run the tests and as expected almost everything is slightly modified.
Without:
Accidental Accidental_Padding Bravura_Reference
With:
Accidental Accidental_Padding Bravura_Current
Without:
Accidental Automatic_Accidentals___C_major_scale_in_Ab Gonville_Reference
With:
Accidental Automatic_Accidentals___C_major_scale_in_Ab Gonville_Current
...

**It looks good**

**Comment by 0xfe** [comment]
Change looks good, Aaron. Can you fix the conflicts please? (Tests are now TS.)

**Comment by AaronDavidNewman** [comment]
:heavy_check_mark: ready to merge.

**Comment by 0xfe** [comment]
Thanks!

**Comment by mscuthbert** [comment]
Small but important change.  Thanks!

**Pull request merged**